docs/guides/mnq_with_nats_terraform_provider.md

```diff
@@ -0,0 +1,122 @@
+---
+page_title: "Using Scaleway Messaging and Queuing service with NATS Terraform provider"
+description: |-
+Using Scaleway Messaging and Queuing service with NATS Terraform provider
+---
+
+# How to use Scaleway Messaging and Queuing config
+
+In this guide you'll learn how to deploy Scaleway Messaging and Queuing config and use the NATS Jetstream provider with,
```

Comment:
This guide shows the combination of Scaleway Messaging and Queuing configuration with the terraform NATS Jetstream provider. Il will allow you to provision and manage NATS Jetstream resources.